deployment/check.yml

```diff
@@ -9,7 +9,10 @@
     deployment: "{{ lookup('env', 'DEPLOYMENT') }}"
     DCs:
       - redis
-      - dist2src-worker
+      - workers
```

Comment:
TBH, I like the singular form better, mainly b/c this is how we have in all of our other projects, and this consistency helps with not having to second guess if this is different or not.